src/shared/mkfs-util.c

```diff
@@ -159,9 +169,8 @@ static int do_mcopy(const char *node, const char *root) {
         if (r < 0)
                 return r;
         if (r == 0) {
-                /* Avoid failures caused by mismatch in expectations between mkfs.vfat and mcopy by disabling
-                 * the stricter mcopy checks using MTOOLS_SKIP_CHECK. */
-                execve(mcopy, argv, STRV_MAKE("MTOOLS_SKIP_CHECK=1"));
+
+                execve(mcopy, argv, envp);
```

Comment:
should be possible to shorten this to `STRV_MAKE("MTOOLS_SKIP_CHECK=1", source_date_epoch)` as it should handle a nul parameter

Reply:
Good one! One question:
Is my assumption correct that STRV_MAKE will not allocate while strv_new will allocate?

Reply:
yes

Reply:
and yeah, strv_new() should not be necessary, just pass this as additional arg to STRV_MAKE()